web/src/state_data.ts

```diff
@@ -30,6 +37,7 @@ export const current_user_schema = z.object({
     is_moderator: z.boolean(),
     is_owner: z.boolean(),
     user_id: z.number(),
+    onboarding_steps: z.array(onboarding_steps_schema),
```

Comment:
Fields in this section are sorted alphabetically, can you move this to the correct position?